install_config/aggregate_logging.adoc

```diff
@@ -1974,6 +1976,51 @@ $ curl --key /etc/elasticsearch/secret/admin-key \
   "https://localhost:9200/project.logging.3b3594fa-2ccd-11e6-acb7-0eb6b35eaee3.2016.06.15"
 ----
 
+[[fluentd-redeploy-certs]]
+== Redeploying EFK Certificates
+
+If you need to redeploy the EFK certificates, first remove the certificates from the `*/etc/origin/logging*` directory 
```

Comment:
Then, re-run

Better yet, you could make these 3 points an ordered list and drop "first" and "then" in the statements.